### DISCUSSION TITLE
Is it possible to use full-text screening in ASReview?

### DISCUSSION THREAD

**Discussion by SagevdBrand** [asreview/asreview#547] [Q&A] [answered]
Can you also use full-text instead of just the abstracts when screening? And would it then also be possible for ASReview to see patterns between correlation tables for example?

**Comment by boer0107-zz** [comment, marked as answer]
It would technically be possible to screen the full-text, but this would mean that you need to have the full-text available in the csv or ris file that you upload in ASReview. This is a problem, because it's difficult to harvest these full-texts automatically for a specific set of publications and also to combine these into one machine-readable file.

We do offer links to the full text, so you can immediately go to a paper and make a decision based on the full text. The software continuously improves its understanding of your decisions based on the title and abstract. But the software can even benefit from decisions that you make based on the full text, because it can detect patterns in title-abstracts even though you as a human weren't able to make a decision based on the title and abstract.
